translations/create_translation_json.py

```diff
@@ -125,9 +131,56 @@ def create_json_file():
     """Create mapping for InDSL and write to file."""
     output_dict = create_mapping_for_translations()
     file_path = "translated_operations.json"
-    with open(file_path, "w") as f:
-        json.dump(output_dict, f, indent=4)
+    # limit the number of keys to 1000
+    output_dict = {k: output_dict[k] for k in list(output_dict.keys())[:1000]}
+    with open(file_path, "w") as file:
+        json.dump(output_dict, file, indent=4)
+
+
+# Compare keys_from_locize.json with the output of create_json_file() and make a new file with the differences
+def compare_and_push_to_locize():
+    """Push differences to locize."""
+    # Pull data from locize
+    pull_url = f"https://api.locize.app/export/{LOCIZE_PROJECT_ID}/latest/en/{NAMESPACE}"
+    headers = {"Authorization": f"Bearer {LOCIZE_API_KEY}", "Content-Type": "application/json"}
+
+    pull_response = requests.get(pull_url, headers=headers, timeout=30)
+    if pull_response.status_code == 200:
+        with open(OUTPUT_FILE, "w") as file:
+            file.write(pull_response.text)
+    else:
+        pull_response.raise_for_status()
+
+    # Get the keys from locize, this is the source of truth
+    with open("keys_from_locize.json", "r") as file:
+        keys_from_locize = json.load(file)
+
+    # Get new keys and values with potential changes from the create_json_file() function
+    with open("translated_operations.json", "r") as file:
+        translated_operations = json.load(file)
```

Comment:
also don't need to load from file. `create_mapping_for_translations` should return that